File: puede-questions/4.cpp
//anagram

#include <iostream>
#include <cstring>
#include <string>
using namespace std;

int main() {
    string kt1;
    string kt2;

    cout << "Kata pertama: ";
    cin >> kt1;
    cout << "Kata kedua: ";
    cin >> kt2;

    cout << anagram(kt1, kt2);
}

string anagram(string kata1, string kata2) {
    if (kata1.length() != kata2.length()) {
        return "Bukan Anagram";
    }


}

File: puede-questions/4.py
#deteksi anagram ~15 menit

def anagram(kata1, kata2):
    if len(kata1) != len(kata2):
        return "Bukan anagram"
    
    for i in kata1:
        if i in kata2:
            kata2.pop(kata2.index(i))
        else:
            return "Bukan anagram"
    
    return "Anagram"

x = list(input("Tuliskan kata pertama: "))
y = list(input("Tuliskan kata kedua: "))

print(anagram(x,y))